_component_design/toggle.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Toggle
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: >
+  A toggle controls how information is formatted on a control panel page.
```

Comment:
A toggle controls the presentation format for information on a control panel page.

Reply:
Done.